src/controllers/simplified-account/settings/switch-psp/switch-to-worldpay/add-flex-credentials.controller.test.js

```diff
@@ -0,0 +1,215 @@
+const ControllerTestBuilder = require('@test/test-helpers/simplified-account/controllers/ControllerTestBuilder.class')
+const { STRIPE } = require('@models/constants/payment-providers')
+const sinon = require('sinon')
+const { formatSimplifiedAccountPathsFor } = require('@utils/simplified-account/format')
+const { paths } = require('@root/routes')
+const Worldpay3dsFlexCredential = require('@models/gateway-account-credential/Worldpay3dsFlexCredential.class')
+
+const ACCOUNT_TYPE = 'live'
+const SERVICE_EXTERNAL_ID = 'service-id-123abc'
+const USER_EXTERNAL_ID = 'user-id-123abc'
+
+const mockResponse = sinon.spy()
+
+const mockWorldpayDetailsService = {
+  check3dsFlexCredential: sinon.stub().resolves(true),
+  update3dsFlexCredentials: sinon.stub().resolves(),
+  updateIntegrationVersion3ds: sinon.stub().resolves()
+}
+
+const {
+  req,
+  res,
+  nextRequest,
+  nextStubs,
+  call
+} = new ControllerTestBuilder('@controllers/simplified-account/settings/switch-psp/switch-to-worldpay/add-flex-credentials.controller')
+  .withServiceExternalId(SERVICE_EXTERNAL_ID)
+  .withAccount({
+    type: ACCOUNT_TYPE,
+    providerSwitchEnabled: true,
+    paymentProvider: STRIPE,
+    allowMoto: true
+    // getSwitchingCredential: getSwitchingCredentialStub
```

Comment:
remove comment or is this a TODO?

Reply:
Whoops. Might as well remove it in the next PR.